nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -379,3 +379,11 @@ def test_plot_connectome_exceptions():
                         adjacency_matrix, node_coords,
                         node_kwargs={'c': 'blue'},
                         **kwargs)
+
+
+def test_singleton_ax_dim():
+    for axis, direction in enumerate("xyz"):
+        shape = [5, 6, 7]
+        shape[axis] = 1
+        img = nibabel.Nifti1Image(np.ones(shape), np.eye(4))
+        plot_stat_map(img, None, display_mode=direction)
```

Comment:
10 loops, best of 3: 234 ms per loop


On Sat, May 9, 2015 at 12:50 PM, Gael Varoquaux <notifications@github.com>
wrote:

> In nilearn/plotting/tests/test_img_plotting.py
> <https://github.com/nilearn/nilearn/pull/576#discussion_r29991782>:
>
> > @@ -379,3 +379,11 @@ def test_plot_connectome_exceptions():
> >                          adjacency_matrix, node_coords,
> >                          node_kwargs={'c': 'blue'},
> >                          **kwargs)
> > +
> > +
> > +def test_singleton_ax_dim():
> > +    for axis, direction in enumerate("xyz"):
> > +        shape = [5, 6, 7]
> > +        shape[axis] = 1
> > +        img = nibabel.Nifti1Image(np.ones(shape), np.eye(4))
> > +        plot_stat_map(img, None, display_mode=direction)
>
>  OK, i did include a test for the cutting function. So i guess i can just
> remove this one, since it's not very DRY.
> I am just asking an estimation of time, to know whether or not to include
> it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/576/files#r29991782>.
>



-- 
DED